src/secure_cookie/session.py

```diff
@@ -233,7 +233,9 @@ def __init__(
     ):
         super(FilesystemSessionStore, self).__init__(session_class=session_class)
 
-        if path is None:
+        if path:
+            os.makedirs(path, exist_ok=True)
```

Comment:
We could use the newer [pathlib.mkdir](https://docs.python.org/3/library/pathlib.html#pathlib.Path.mkdir) instead of the `os` module. 

Also, **personally**, I would use a `try/catch` here so the fact that we are aware of the exception and yet chose to raise it anyway (since there isn't much we can do in this case for `OSError`s) is more explicit as opposed to let it look like just a unhandled exception.

That's what we do in `cachelib.FilesystemCache` ([take a look](https://github.com/pallets/cachelib/blob/main/src/cachelib/file.py#L47)).  But replace the if statement with `exist_ok` like you're already doing.

```python
try:
    Path.mkdir(parents=True, exist_ok=True)
except OSError:
        raise
```

Reply:
I modified the code to use `pathlib.Path.mkdir()` in a `try except`.

As the file imports `pathlib` now, we could also remove [`path.join()` call](https://github.com/sblondon/secure-cookie/blob/50a20b24af6ca5a3c909f23fa0e2c44203732142/src/secure_cookie/session.py#L263) and the `os.path` import. I can do that in another PR if you want.

Reply:
Yeah, that would be nice. I suppose we can go ahead and do it in this PR since it's related @sblondon 👍🏻

Reply:
It's pushed

Reply:
Good work @sblondon ! Could you please add a new entry in `CHANGELOG.rst` describing the change? Something simple like "`FilesystemSessionStore `will now attempt to create `path` if the specified location does not exist" will do.

Reply:
@northernSage Done (with a lightly simpler sentence) I added the skeleton for the next release too.